BackendTaskPilot/Program.cs

```diff
@@ -0,0 +1,38 @@
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+var builder = WebApplication.CreateBuilder(args);
+var app = builder.Build();
```

Comment:
Only these methods are good candidates for the use of a Program (or Startup class). The rest should be extracted in their separate folders and classes.